randovania/data/json_data/prime2.txt

```diff
@@ -2756,8 +2752,8 @@ Asset id: 1272952761
       After Transport Center Gate
   > Transport Center/Event - Transport Center Gate
       Any of the following:
-          Bomb Slot without Bombs (Space Jump)
           Morph Ball and Morph Ball Bomb
+          No Space Jump Boots and Bomb Slot without Bombs (Space Jump)
```

Comment:
No Space Jump Boots?

Reply:
Yes because space jump isn't needed. I did this because the nsj template includes sunburst, which isn't viable for most slots. Better solution might be to make other template or something.

Reply:
`No Space Jump Boots` implies that you *can't* have SJ.

Reply:
So what do you recommend?